package.json

```diff
@@ -12,7 +12,7 @@
   },
   "dependencies": {
       "pkginfo": "~0.3.0",
-      "underscore": "~1.5.2"
+      "underscore": "1.8.3"
```

Comment:
Yes, this is needed in [test code](https://github.com/Lewuathe/readonly/blob/master/test/readonly-test.js).